songsuro/condition/model.py

```diff
@@ -54,7 +61,14 @@ def forward(self, lyrics, quantized_f0):
 			# + timbre_embedding + style_embedding
 		)
 
-		summation_lengths = torch.tensor([summation_embedding.shape[2]])
+		summation_lengths = torch.cat(
+			(
+				lyrics_lengths,
+				# quantized_f0_lengths,
+				# timbre_lengths,
+				# style_lengths
+			)
+		)
```

Comment:
I think you must not concat the lengths and put it in the enhanced condition encoder.
Ummmm... I bet we have to read the literature about enhanced condition encoder and find out what will be the best strategy to put the embeddings (max four embeddings)
But, I am pretty sure that concat original lengths is not the case.
Because the summation_embedding dimension will be [Batch, hidden size, length] right? 
But the summation_lengths dimension will be [4, Batch] (if we use all four embeddings)
The input length tensor might be size of the [Batch] (1-d tensor)